crawl-ref/source/dat/des/arrival/large.des

```diff
@@ -476,8 +476,8 @@ MAP
 .................@.................
 .cccccc...cccccc...cccccc...cccccc.
 .c....c...c....c...c....c...c....c.
-.c.(..c...c.{..c...c..[.c...c..>.c.
-.c....cc+cc....cc+cc....cc+cc....c.
+.c.(..cc+cc.{..cc+cc..[.cc+cc..>.c.
+.c....+...+....+...+....+...+....c.
```

Comment:
I think instead of re-shaping these small rooms, a `KPROP: no_monster_gen` for the first part of this pyramid (the entry rooms and their anterooms) is a better solution.